### PR TITLE
add view home

### PR DESCRIPTION
**Pull request by NamBZ**
- Add home layout
- Use bower to manage Font-Awesome package
- Fix Locale vi for Authentication

### CODE REVIEW

**Comment by NamBZ** [comment]
ready

File: .bowerrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "directory": "public/bower_components/"
+}
```
Comment (huyenttm-0901) [inline .bowerrc]: eof

File: resources/views/layouts/home.blade.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+        <!-- Fonts -->
+        <link rel="stylesheet" href="{{ asset('bower_components/font-awesome/css/font-awesome.min.css') }}">
+
+        <!-- Styles -->
+        <link rel="stylesheet" href="{{ asset('css/style.css') }}">
+    </head>
+    <body>
+        <div class="header" style="text-align:center;padding:10px">
+            <a href="/" title="{{ config('app.name', 'My Blog') }}" id="logo">
```
Comment (huyenttm-0901) [inline resources/views/layouts/home.blade.php]: ko dùng url

**Comment by NamBZ** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
merge

**Pull request merged**